AdblockPlusSafari/NSString+MarkdownRenderer.h

```diff
@@ -0,0 +1,24 @@
+//
```

Comment:
@dedecej seems unrelated to me, doesn't seem to be used. Or am I missing something?